pages.pt_BR/common/bup.md

```diff
@@ -0,0 +1,24 @@
+# bup
+
+> Sistema de backup baseado no formato Git packfile, oferecendo salvamentos incrementais e desduplicação global.
+> Mais informações: <https://github.com/bup/bup>.
+
+- Inicializa um repositório de backup no diretório local especificado:
+
+`bup -d {{caminho/para/repositório}} init`
+
+- Prepara um determinado diretório antes de fazer um backup:
+
+`bup -d {{caminho/para/repositório}} index {{caminho/para/diretório}}`
+
+- Faz o backup de um diretório para o repositório:
+
+`bup -d {{caminho/para/repositório}} save -n {{nome_do_backup}} {{caminho/para/diretório}}`
```

Comment:
The following example fix can be applied to the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/bup.md) page and then here:

```suggestion
- Faz o backup de um [d]iretório para o repositório:

`bup -d {{caminho/para/repositório}} save -n {{nome_do_backup}} {{caminho/para/diretório}}`
```